build.gradle.kts

```diff
@@ -18,30 +18,32 @@ repositories {
 dependencies {
     implementation(kotlin("stdlib-jdk8"))
     implementation(kotlin("reflect"))
-    implementation("org.springframework.boot:spring-boot-starter-web:2.2.6.RELEASE")
-    implementation("org.springframework.boot:spring-boot-starter-actuator:2.2.6.RELEASE")
+    val springBootVersion = "2.2.7.RELEASE"
+    implementation("org.springframework.boot:spring-boot-starter-web:$springBootVersion")
+    implementation("org.springframework.boot:spring-boot-starter-actuator:$springBootVersion")
     implementation("com.fasterxml.jackson.module:jackson-module-kotlin:2.11.0")
-    implementation("org.springframework.boot:spring-boot-starter-webflux:2.2.6.RELEASE")
+    implementation("org.springframework.boot:spring-boot-starter-webflux:$springBootVersion")
 
     // swagger
-    val springdocVersion = "1.3.4"
+    val springdocVersion = "1.3.9"
     implementation("org.springdoc:springdoc-openapi-ui:$springdocVersion")
     implementation("org.springdoc:springdoc-openapi-kotlin:$springdocVersion")
 
     // spring data
-    implementation("org.springframework.data:spring-data-commons:2.2.6.RELEASE")
+    implementation("org.springframework.data:spring-data-commons:2.3.0.RELEASE")
```

Comment:
if youre bumping data should we just bump the rest of the spring deps to 2.3.0? Also not a blocker and we could just do it in a diff PR

Reply:
At the time I updated, the gradle dependencies plugin only suggested it as it was. I bumped all of them now.